vllm/worker/model_runner.py

```diff
@@ -659,6 +659,16 @@ def _prepare_model_input_tensors(
                                            dtype=torch.long,
                                            device=self.device)
 
+        logits_soft_cap = getattr(self.model_config.hf_config,
+                                  'final_logit_softcapping', None)
+        if logits_soft_cap is not None and self.attn_backend.get_name(
+        ) != "flashinfer":
```

Comment:
Could I check if `logits_soft_cap` is supposed to be the `attn_logit_softcapping` value instead? The two values are different in the Gemma2 config.
```
"attn_logit_softcapping": 50.0,
"final_logit_softcapping": 30.0,
```

Reply:
@yongxb Nice catch! `final_logit_softcapping` is used to cap the final logits before sampling. @LiuXiaoxuanPKU Could you please fix this?